libraries/smarty/Config_File.class.php

```diff
@@ -141,88 +162,6 @@ function get($file_name, $section_name = NULL, $var_name = NULL)
         }
     }
 
-
```

Comment:
I'm concerned that we remove stuff in the smarty library, something we shouldn't touch afaik